core/src/components/checkbox/test/indeterminate/checkbox.e2e.ts

```diff
@@ -0,0 +1,11 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('checkbox: indeterminate', () => {
+  test('should not have visual regressions', async ({ page }) => {
```

Comment:
Does indeterminate state have RTL-specific logic? If not, we can probably disable the rtl check